web/server/codechecker_server/api/report_server.py

```diff
@@ -623,11 +623,26 @@ def get_report_details(session, report_ids):
         extended_data.filePath = file_path
         extended_data_list[report_id].append(extended_data)
 
+    # Get Comments for report data
+    comment_data_list = defaultdict(list)
+    comment_query = session.query(Comment, Report.id, File.filepath)\
+        .outerjoin(Report, Report.bug_id == Comment.bug_hash) \
+        .outerjoin(File,
+                   File.id == Report.file_id)
+
+    for data, report_id, file_path in comment_query:
+        report_id = report_id
+        comment_data = comment_data_db_to_api(data)
+        comment_data.filePath = file_path
```

Comment:
Why do you try to set a field why doesn't even exists? The CommentData struct doesn't have a filePath field:
https://github.com/Ericsson/codechecker/blob/47cf966edcafaaaf1047298d8fe0b03f0f3ca82a/web/api/report_server.thrift#L303-L309

Comments are connected to Reports through the Report hash (bug id) field, and every Report points to a file path. So I think there is no need to query the file path and use it here.

Reply:
Yes, that totally makes sense. I am removing the file path line